review/decorators.py

```diff
@@ -1,4 +1,4 @@
-from django.utils.six import wraps
+from six import wraps
```

Comment:
We probably don't need `six` anymore since we don't support Python 2.x. This is easy to fix later if this works at the moment.

https://six.readthedocs.io/
> `@six.wraps(wrapped, assigned=functools.WRAPPER_ASSIGNMENTS, updated=functools.WRAPPER_UPDATES)`
>
> This is Python 3.2’s functools.wraps() decorator. It sets the __wrapped__ attribute on what it decorates. It doesn’t raise an error if any of the attributes mentioned in assigned and updated are missing on wrapped object.